### PR TITLE
[CLOUD-2328] Update JWS Openshift images to JWS 3.1 SP2

### PR DESCRIPTION
**Pull request by bekionel**
Signed-off-by: Sokratis Zappis <szappis@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: tomcat7/image.yaml [hunk 1]
```diff
@@ -66,6 +66,9 @@ artifacts:
     - path: jws-application-servers-3.1.1-RHEL7-x86_64.zip
       description: "Artifact is available on the Customer Portal: https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=webserver&downloadType=securityPatches&version="
       md5: a74c29dfc2f5431e5b106a8932745092
+    - path: jws-application-servers-3.1.2-RHEL7-x86_64.zip
```
Comment (goldmann) [inline tomcat7/image.yaml]: How is it installed? You did not update the install scripts.

File: tomcat7/image.yaml [hunk 2]
```diff
@@ -66,6 +66,9 @@ artifacts:
     - path: jws-application-servers-3.1.1-RHEL7-x86_64.zip
```
Comment (goldmann) [inline tomcat7/image.yaml]: Is 3.1.1 required to apply 3.1.2?

**Review by goldmann — CHANGES_REQUESTED** [review]

File: tomcat7/image.yaml (hunk 2 above)
Comment (bekionel) [reply tomcat7/image.yaml]: From the information I have, this is a cumulative patch which means 3.1.1 is not required, so I made the necessary changes.

File: tomcat7/image.yaml (hunk 1 above)
Comment (bekionel) [reply tomcat7/image.yaml]: After the changes it will be unzipped on top of 3.1.0 as of modules/jws-unpack-tomcat7/run

**Review by goldmann — CHANGES_REQUESTED** [review]
The jws-application-servers-3.1.2-RHEL7-x86_64.zip file is not used.

**Review by goldmann — APPROVED** [review]

**Comment by goldmann** [comment]
It was my issue, I didn't found that the install script can work with any micro release. Approved and merged!

**Pull request merged**